app/views/case_contacts/form/details.html.erb

```diff
@@ -24,15 +24,14 @@
         <h5 classs="mb-3"><label>a. Was contact made?</label></h5>
         <div class="form-check radio-style mb-20">
           <%= form.radio_button :contact_made, true,
-            checked: @case_contact.contact_made,
+            checked: true,
```

Comment:
I don't think we need this line if we are setting `contact_made: true` in the controller.

Reply:
That is true, thanks for catching it!